### ISSUE TITLE
Gnome terminal is closing when try source theme

### ISSUE THREAD

**Issue by niltonvasques** [chriskempson/base16-gnome-terminal#8]
I'm on debian jessie.
The version of GNOME Terminal that I'm using is  3.14.1

This issues happen when I run any base16 theme with this command:

`source base16-default.dark.sh`

And when I'm open gnome terminal again, I can't see theme a profile installed.

**Comment by kusold** [comment]
Does it show up if on the menu you do:
`Terminal > Change Profile`

I was having problems because I was expecting this to show up as a color pallette/scheme.

**Comment by niltonvasques** [comment]
I try it here, but only Default profile is available.

What version of Gnome Terminal you are using?

**Comment by kusold** [comment]
`3.10.2` -- You are a bit newer than me but I plan on updating to Fedora 22 soon (which has 3.16) and I'll retest this.

**Comment by willisrocks** [comment]
I'm having the same issue on Fedora 22.

**Comment by 190n** [comment]
I have this too. Same setup as @niltonvasques - Debian 8, Gnome Terminal 3.14.1.

**Comment by benbenolson** [comment]
I've got the same issue. When I source the scripts, gnome-terminal simply closes. Upon reopening, nothing has changed, and no new color schemes are available in the settings.  I'll inspect further.

**Comment by ksdev-pl** [comment]
Try to install `uuid-runtime` before running the scripts, at least on Debian Jessie - it helped in my case.

**Comment by benbenolson** [comment]
I've tried that already, and it still doesn't work. I've just now tried it on a fresh install of Debian Jessie, after selecting "GNOME" in the installer.  Default repositories.

**Comment by benbenolson** [comment]
I got it to work. By default, my terminal came with one profile, called "Unnamed". This meant that there was no "default" key in dconf, and so there were no profiles at all. Therefore, the script was unable to find the default profile and append to it.

So, what I did was create a new profile, name it something else, and then delete the "unnamed" profile. I thought that this would be enough, but GNOME's terminal was persistent-- looking in dconf-editor, I still didn't see any default profiles.  So, I went back to my new profile and changed some arbitrary setting, and then set that profile as the default (or, "profile to use in a new terminal").

Confirming in dconf-editor, I now had a few profiles, including one default one. I then sourced the script, and it crashed gnome-terminal still, but this time, upon restarting the application, I was able to choose the new profile.

Success!

**Comment by 190n** [comment]
@benbenolson That worked for me too.

**Comment by faest** [comment]
@benbenolson You are a legend, this worked for me as well!

**Comment by auspex-nl** [comment]
Another note, the readme should be updated to state that both dconf & uuidgen need to be available. On debian 8, I did have at least one custom profile before running the script but did not have the 'uuid-runtime' package installed and thus did not have uuidgen. The profiles were made but with their text-names, next time running gnome terminal no profiles could be selected (my own custom profiles were gone too). I had to remove the invalid entries from the list using dconf-editor:

```
/org/gnome/terminal/legacy/profiles: 
```

Profiles work fine now.
